Specs/UI/SajithKumar-spec.md

```diff
@@ -0,0 +1,117 @@
+- And when we need to send the message to the processing module, we will create the object and send the message using the functions present in it.
+
+### JavaFX and Scene Builder
+- To desing the GUI of a desktop application there are many tools available eg : JavaFX, Swing, AWT etc.
+- Currently we would be using javaFX for adding shapes to our canvas.
+- **JavaFX** is a set of graphics and media packages that enables developers to design, create, test, debug, and deploy rich client applications that operate consistently across diverse platforms.
+- JavaFX is the successor of Swing which includes FXML and CSS files which eases the designing and formatting of the web pages.
+- **JavaFX Scene Builder** is a visual layout tool that lets users quickly design JavaFX application user interfaces, without coding. Users can drag and drop UI components to a work area, modify their properties, apply style sheets, and the FXML code for the layout that they are creating is automatically generated in the background. The result is an FXML file that can then be combined with a Java project by binding the UI to the application’s logic.
+- We would be using javaFX Scene Builder to create the view of shapes to select.
+- Scene Builder will be beneficial because we can add various options and features of UI (eg: Button, Checkbox etc... ) easily.
+
+
+### FrameWork : MVC Pattern
+
+-  Model
+    The Model describes the business logic and it is characterized by a set of classes. It works on to design business rules for data on how the data is changed or handled.
+- View
+    The View here represents the UI components. The View displays the data which the controller sends back as a form of the result. The Model also can be converted into the UI using View.
+
+- Controller
+    For proceeding incoming requests, the controller is highly responsive. Through the Model to View, the Controller gets the data of the user. Between the Model and the View, the Controller acts as a facilitator.
+
+
+    
+- Here, the view will be built using javaFX scene builder. It will be a fxml file. CSS properties can be used for providing various attributes like size of text area, colour etc...
+
+### Shapes interface
+- On click provides an menu of standard shapes like circle, rectangle, square and line segment.
+- Also provides the option to select the colour of the shape.
+- Any shape can be drawn on the canvas by dragging the mouse after selecting the shape. 
+``` java
+public interface IDrawShapes {
+    // parameter: Pixel start - position of mouse drag started with RGB values
+    //                 Pixel end -  position of mouse drag ended with RGB values
+    void draw_circle(Pixel start, Pixel end);
+    void draw_square(Pixel start, Pixel end);         
+    void draw_rectangle(Pixel start, Pixel end); 
+    void draw_line(Pixel start, Pixel end);        
+    void draw_oval(Pixel start, Pixel end);    
+    void getSnapshot(Pixel start, Pixel end);
+}

```

Comment:
Please change the comment styles and method names according to the coding guidelines.

Reply:
I will send a new PR for that Spec. I will follow the provided guidelines from next time. Thank you.